rpc/call_test.go

```diff
@@ -38,12 +38,12 @@ func TestCall(t *testing.T) {
 			{
 				FunctionCall: FunctionCall{
 					// ContractAddress of predeployed devnet Feetoken
-					ContractAddress:    utils.TestHexToFelt(t, "0x49d36570d4e46f48e99674bd3fcc84644ddd6b96f7c741b1562b82f9e004dc7"),
-					EntryPointSelector: utils.GetSelectorFromNameFelt("name"),
+					ContractAddress:    utils.TestHexToFelt(t, DevNetETHAddress),
+					EntryPointSelector: utils.GetSelectorFromNameFelt("decimals"),
```

Comment:
I had some issues related to felt.Felt comparison using the "name" method. Using "decimals" was simpler to compare